### ISSUE TITLE
Diffirent loss between paper and code

### ISSUE THREAD

**Issue by neiheih** [SenJia/EML-NET-Saliency#1]
Hi,
In your paper , the loss is a hybrid loss consist of nss, cc and kld. But in the code ,you simply use mse  as loss. Is there any difference between the experimental results of the two losses? Can you share the code of the hybrid loss?
tks!

**Comment by SenJia** [comment]
Hi, I have uploaded the loss file.

**Comment by samyak0210** [comment]
Are you training your model using the new loss function or MSE loss function?
